sphinxcontrib/websupport/builder.py

```diff
@@ -178,6 +178,25 @@ def handle_finish(self):
                     shutil.rmtree(dst)
                 shutil.move(src, dst)
 
+        # copy resource files to static dir
+        resources = [
```

Comment:
It would be nice to define this as a constant.

Reply:
I got it. I'll fix it.

Reply:
I fixed it.